pkg/manager-nvme/manager.go

```diff
@@ -563,6 +623,10 @@ func (s *Storage) recoverVolumes() error {
 	return nil
 }
 
+func (s *Storage) CleanupNamespaces(namespacesToKeep []nvme.NamespaceIdentifier) error {
```

Comment:
this can be private `cleanupNamespaces`, right?

Reply:
or just call `storage.purgeExcept` directly from `CleanupVolumes`

Reply:
I ended up just calling it directly.